lasagne/objectives.py

```diff
@@ -70,8 +95,9 @@ def __init__(self, input_layer, loss_function=mse):
         self.input_layer = input_layer
         self.loss_function = loss_function
```

Comment:
The docstring for the `loss_function` argument needs to be updated to account for the new argument.